FEATURES.md

```
@@ -68,6 +68,11 @@ Status guide: `[shipped]` is live today, `[in-progress]` is partially implemente
 
 ## Combat & Gameplay
 
+### 🔫 Grenade Hold-to-Throw `[shipped]`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)